src/main/java/cz/idomatojde/entity/Offer.java

```diff
@@ -0,0 +1,138 @@
+package cz.idomatojde.entity;
+
+import com.sun.istack.NotNull;
```

Comment:
I would avoid non-standard imports.
@Column(nullable = false) or the javax.validation.constraints.NotNull seems better.